<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批量重命名文件</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">backround = rgb(255,255,255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filePathEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="getPathButton">
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="sourceFileList"/>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="needDate">
         <property name="text">
          <string>当前日期</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="needTime">
         <property name="text">
          <string>当前时间</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="needNumber_2">
         <property name="text">
          <string>自动序号</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="needOriginalFileName">
         <property name="text">
          <string>原文件名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="OriginFileNameEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="needFirstCustomText">
         <property name="text">
          <string>自定义字符串1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FirstCustomTextEdit"/>
       </item>
       <item>
        <widget class="QCheckBox" name="needSecondCustomText">
         <property name="text">
          <string>自定义字符串2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="secondCustomTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,8">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>示例文本：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="exampleFileName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="reNameFileButton">
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string>Do It!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="resultFileList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionTutorial">
   <property name="text">
    <string>Tutorial</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>getPathButton</tabstop>
  <tabstop>sourceFileList</tabstop>
  <tabstop>needDate</tabstop>
  <tabstop>needTime</tabstop>
  <tabstop>needNumber_2</tabstop>
  <tabstop>needOriginalFileName</tabstop>
  <tabstop>needFirstCustomText</tabstop>
  <tabstop>needSecondCustomText</tabstop>
  <tabstop>reNameFileButton</tabstop>
  <tabstop>resultFileList</tabstop>
  <tabstop>OriginFileNameEdit</tabstop>
  <tabstop>FirstCustomTextEdit</tabstop>
  <tabstop>secondCustomTextEdit</tabstop>
  <tabstop>filePathEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
